examples/pytorch/README.md

```diff
@@ -199,6 +198,7 @@ You can easily log and monitor your runs code. The following are currently suppo
 * [TensorBoard](https://www.tensorflow.org/tensorboard)
 * [Weights & Biases](https://docs.wandb.ai/integrations/huggingface)
 * [Comet ML](https://www.comet.ml/docs/python-sdk/huggingface/)
+* [Neptune](https://docs.neptune.ai)
```

Comment:
we should link to HF integration page.

Reply:
Yes, if we can have it ready in time (the page doesn't exist yet)

Reply:
But do we have any expected path? `https://docs.neptune.ai/integrations-and-supported-tools/model-training/huggingface`? Even if it will be non-existent it's okay for now. I don't think we'll be able to easily update it later as it's in an external repository.

Reply:
Yes, this exactly was my concern, especially with new docs (and URLs) coming. But we'll need to set up redirects anyway, so I think I can create a placeholder in the current docs that should be more useful than nothing :)